Fixed 'alt text was rendering with inline image in pdf' issue

anuradhas248 opened the pull request:
Fixed 'alt text was rendering with inline image in pdf' issue and also allowed http with https for image url.

Jira ID - KMS-4389
Git Issue - [505](https://github.com/opensalt/opensalt/issues/505)

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/opensalt/opensalt/506)
<!-- Reviewable:end -->

roverwolf commented on src/Controller/PdfExportController.php:
```diff
@@ -69,8 +69,8 @@ function () use ($phpWordObject) {
     {
-        $pattern = '/\!\[([^\]]*)\]\(((?:https:\/\/|\/)[^\)]+)\)/';
-        $result = preg_replace($pattern, '$1 <img src = "$2">', $string);
+        $pattern = '/\!\[([^\]]*)\]\(((?:(?:https?|):\/\/|\/)[^\)]+)\)/';
```
What was the intention of this change?  Do we really want to allow `://something-here` to be valid?  Or was the intention to just add the `?` after the `s` to allow `http`?

roverwolf replied:
Or was the pattern supposed to change at all?  Just the replacement (removing $1)?

anuradhas248 replied:
Testers have found few urls that used pattern ![alt text](//something-here). And http was not allowing in previous pattern. So for allow http and pattern similar like "://something-here." I updated pattern. At the same time I removed $1 as it was showing first bracket text along with image.

anuradhas248 commented:
@roverwolf - As per your suggestions, I have updated pattern that allowing http(as some of content used http url to render image) along with https. And removed second condition which was allowing :// prefix.
Please verify.
Thank you.
